src/compiler/llvm/mir.rs

```diff
@@ -520,7 +520,9 @@ impl MirTypeDef {
                     .iter()
                     .map(|f| {
                         p.get_type(f.ty)
-                            .expect("Cannot find given Type ID")
+                            .unwrap_or_else(|e| {
+                                panic!("Cannot find given Type ID {:?} in\n{:?}", e, p.ty_table)
+                            })
```

Comment:
Add better formatting for the table.

Reply:
Should this be panicking?

Reply:
yes, because by this time, any type referenced in this type definition should already be added (or in the case of structures declared).